<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CZMLPiechartDialogBase</class>
 <widget class="QDialog" name="CZMLPiechartDialogBase">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>367</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>403</width>
    <height>268</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>403</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>3D Piechart</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>prism.png</normaloff>prism.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QComboBox" name="layerList">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select layer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output file:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFileName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBrowse">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select attributes for slices</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertical scale</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbLin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Linear</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbSqrt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Square root</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbLog">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Logarithmic</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current range (meters):</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRange">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Factor:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scaleFactor">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Colors:</string>
   </property>
  </widget>
  <widget class="QgsColorButton" name="cb1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>32</width>
     <height>27</height>
    </rect>
   </property>
   <property name="color">
    <color>
     <red>0</red>
     <green>0</green>
     <blue>127</blue>
    </color>
   </property>
  </widget>
  <widget class="QgsColorButton" name="cb2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>32</width>
     <height>27</height>
    </rect>
   </property>
   <property name="color">
    <color>
     <red>255</red>
     <green>255</green>
     <blue>255</blue>
    </color>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbAddName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>add names from attribute:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="strAttrList">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbCreateLegend">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Create HTML legend - attribute name:</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLegendAttrName">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbLegendSettings">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Legend settings...</string>
   </property>
  </widget>
  <widget class="QListWidget" name="attrList">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QPushButton</extends>
   <header>qgscolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
